botorch/utils/sampling.py

```diff
@@ -250,7 +250,7 @@ def sample_polytope(
     d = x0.size(0)
     # uniform samples from unit ball in d dims
     Rs = sample_hypersphere(
-        d=d, n=n_tot, dtype=A.dtype, device=A.device, seed=seed
+        d=d, n=n_tot, dtype=A.dtype, device=A.device, seed=seed + 1
```

Comment:
Would you mind adding a brief comment for why this is necessary, that links to https://github.com/pytorch/botorch/issues/2156 please?

Reply:
Done. Hope it is clearer now 🙂